include/algorithms/math/logistic.h

```diff
@@ -89,15 +96,19 @@ class DAAL_EXPORT Batch : public daal::algorithms::Analysis<batch>
     typedef algorithms::math::logistic::Input  InputType;
     typedef algorithms::math::logistic::Result ResultType;
 
-    /** Default constructor */
-    Batch()
+    /**
+     * Default constructor
+     * \DAAL_DEPRECATED
+     */
+    DAAL_DEPRECATED Batch()
     {
         initialize();
     }
 
     /**
      * Constructs the logistic function by copying input objects of another logistic function
      * \param[in] other function to be used as the source to initialize the input objects of the logistic function
+     * \DAAL_DEPRECATED_USE{ cloneImpl() }
```

Comment:
cloneImpl is deprecated also, just DAAL_DEPRECATED

Reply:
fixed